virtcontainers/fc.go

```diff
@@ -250,7 +263,10 @@ func (fc *firecracker) createSandbox(ctx context.Context, id string, networkNS N
 
 	fc.vmPath = filepath.Join(fc.chrootBaseDir, hypervisorName, fc.id)
 	fc.jailerRoot = filepath.Join(fc.vmPath, "root") // auto created by jailer
-	fc.socketPath = filepath.Join(fc.jailerRoot, fcSocket)
+
+	// Firecracker and jailer automatically creates default API socket under /run
+	// in the name of "firecracker.socket"
```

Comment:
in the name? or with the name? :thinking:

Reply:
lol,,,I'll follow your suggestion. never really figuring out those prepositional phrases. 🤯😖